src/create-client.ts

```diff
@@ -40,14 +40,9 @@ export const clientConstructor = ({
         },
       })
         .then((response) => response.data)
-        .catch((error) => {
-          if (
-            error &&
-            error.response &&
-            error.response.data &&
-            error.response.data.description
-          ) {
-            throw new Error(error.response.data.description)
+        .catch((error: any) => {
+          if (error && error.response && error.response.data) {
+            throw new Error(error.response.data)
           } else {
             throw new Error("Couldn't make request")
           }
```

Comment:
The `Error` constructor accepts string, not an object. 

@itaied246 what data are you missing in the error?  If it is the code, we could add it to the message:
```suggestion
          if (error?.response?.data) {
            const { code, description } = error.response.data
            if (description) {
              throw new Error(`${code ? `${code}: ` : ''}${description}`)
            }
          }
          throw new Error("Couldn't make request")
```